app/Http/Controllers/InstitutionsController.php

```diff
@@ -61,10 +61,11 @@ public function update(HttpRequest $request, $id)
 
         $userInstitutions = array_column($userInstitutions, 'institution_id');
         if (in_array($institutionId, $userInstitutions)) {
-            
+            School_additional_data::CreateOrUpdate($additional_data);
```

Comment:
We not yet implemented such class yet.Add a TODO comment and remove this method

Reply:
Done